mysql-test/suite/tianmu/t/issue781.test

```diff
@@ -0,0 +1,76 @@
+--source include/have_tianmu.inc
+
+USE test;
+
+--disable_warnings
+
+## DDL
+
+DROP TABLE IF EXISTS cs1;
+
+CREATE TABLE cs1 (d1 int, d2 int) ENGINE=TIANMU;
+
+## insert data
+
+INSERT INTO cs1 VALUES (125, 125);
+INSERT INTO cs1 VALUES (-125, -125);
+INSERT INTO cs1 VALUES (77, 55);
+INSERT INTO cs1 VALUES (33, 66);
+INSERT INTO cs1 VALUES (99998, 998);
+INSERT INTO cs1 VALUES (99999, 999);
+
+## select before update
+
+SELECT * FROM cs1;
+
+##  update with in of more than one elem
+
+UPDATE cs1 SET d1=888, d2=777 WHERE d1 in (125, 99999);
+
+## select again
+
+SELECT * FROM cs1;
```

Comment:
Adds a case for `json` type. Due to the json is a type used to stored array data in mysql. Now that, the json is not supported by Tianmu. 

create table tb (c1 json, c2 int) engine =tianmu;